app/src/main/java/com/kodeco/android/coordplot/HorizontalSlider.kt

```diff
@@ -0,0 +1,15 @@
+package com.kodeco.android.coordplot
+
+import androidx.compose.material.Slider
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+
+@Composable
+fun createHorizontalSlider(modifier: Modifier = Modifier, value : Float =0.5f, changeValue : (Float) -> Unit){
```

Comment:
Nice work on creating component composable functions! One thing to note, which is typically not the norm for naming convention with functions, but _composable_ functions should begin with a capital letter.  Just like classes do.  

<img width="629" alt="Screen Shot 2023-09-05 at 9 13 00 PM" src="https://github.com/sagar-jh/coordplot/assets/2581931/9d7687e1-6348-4d25-b824-58e70b8e6faa">